.changeset/good-rocks-appear.md

```diff
@@ -0,0 +1,19 @@
+---
+'@graphql-hive/core': minor
+---
+
+Add support for providing a target for usage reporting with organization access tokens.
+This can either be a slug following the format "$organizationSlug/$projectSlug/$targetSlug" (e.g "the-guild/graphql-hive/staging")
```

Comment:
```suggestion
This can either be a slug following the format `$organizationSlug/$projectSlug/$targetSlug` (e.g `the-guild/graphql-hive/staging`)
```